client/__tests__/actions/signinActions.test.js

```diff
@@ -0,0 +1,59 @@
+/* global window */
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import expect from 'expect';
+import * as actions from '../../actions/authActions';
+import * as types from '../../actions/types';
+import mockLocalStorage from '../../../__mocks__/mockLocalStorage';
+
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+window.localStorage = mockLocalStorage;
+
+describe('Sign in action', () => {
+  beforeEach(() => moxios.install());
+  afterEach(() => moxios.uninstall());
+  const userData = {
+    username: 'louis',
+    password: 'password'
+  };
+
+  it('contains a logout function', () => {
+    expect(actions.logout()).toBeA('function');
```

Comment:
The information in your `it` function is kinda incomplete. I guess you wanted to do `it(should contains a logout function)`